recipes/immer/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+from conans import ConanFile, tools, errors
+
+
+class ImmerConan(ConanFile):
+    name = "immer"
+    license = "BSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/arximboldi/immer"
+    description = "Postmodern immutable and persistent data structures for C++---value semantics at scale"
+    topics = ("header", "header-only", "persistent", "modern", "immutable",
+              "data structures", "functional", "value semantics", "postmodern",
+              "rrb-tree")
+    no_copy_source = True
+    settings =  "compiler"
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 14
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True,
+                  destination=self._source_subfolder)
+
+    def package(self):
+        include_folder = self._source_subfolder
+        self.copy(pattern="*.hpp", dst="include", src=include_folder)
+        self.copy(pattern="LICENSE", dst="licenses", src=include_folder)
+
+    def package_id(self):
+        self.info.header_only()
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "8",
+            "clang": "8",
+        }
```

Comment:
```suggestion
        # Reference: https://en.cppreference.com/w/cpp/compiler_support/14
        return {
            "apple-clang": "5.1",
            "clang": "3.4",
            "gcc": "5",
            "intel": "17",
            "sun-cc": "5.15",
            "Visual Studio": "15"
        }
```

I suggest using these versions instead. Taken from [another recipe](https://github.com/conan-io/conan-center-index/blob/master/recipes/rmlui/3.3/conanfile.py).